sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/NullPointerDereference.CSharp9.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
```

Comment:
No test case with `new()`?

Reply:
I added parameterless overload to `TargetTypedNew`